test/statetest/statetest_export.cpp

```diff
@@ -49,6 +49,8 @@ namespace evmone::test
         return "TR_BLOBVERSION_INVALID";
     case BLOB_GAS_LIMIT_EXCEEDED:
         return "TR_BLOBLIST_OVERSIZE";
+    case EOF_CREATION_TRANSACTION:
+        return "TR_EOFCreationTransaction";
```

Comment:
Use pytest exception name, e.g. `EOFException.EOFCreationTransaction`. They map this by default.

Reply:
changed